pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -188,97 +188,81 @@ func buildTransport(mesh *meshconfig.MeshConfig, node *model.Proxy, uid attribut
 	}
 
 	res := &mccpb.TransportConfig{
-		CheckCluster:  model.BuildSubsetKey(model.TrafficDirectionOutbound, "", model.Hostname(policy), port),
-		ReportCluster: model.BuildSubsetKey(model.TrafficDirectionOutbound, "", model.Hostname(telemetry), port),
+		CheckCluster:      model.BuildSubsetKey(model.TrafficDirectionOutbound, "", model.Hostname(policy), port),
+		ReportCluster:     model.BuildSubsetKey(model.TrafficDirectionOutbound, "", model.Hostname(telemetry), port),
+		NetworkFailPolicy: &mccpb.NetworkFailPolicy{Policy: mccpb.FAIL_CLOSE},
+		// internal telemetry forwarding
+		AttributesForMixerProxy: &mpb.Attributes{Attributes: attributes{"source.uid": attrUID(node)}},
 	}
-	// TODO(yangminzhu): remove this after the default on client code is changed.
-	// So far there is no use case I know where policy check would fail open
-	res.NetworkFailPolicy = &mccpb.NetworkFailPolicy{Policy: mccpb.FAIL_CLOSE}
 
-	// Those settings are not backward compatible.
-	if has10Config(node) {
-		// internal telemetry forwarding
-		res.AttributesForMixerProxy = &mpb.Attributes{Attributes: attributes{"source.uid": uid}}
+	// These settings are not backward compatible with 0.8.
```

Comment:
Please add a comment saying proxy version is found only in and after 1.0.0.

Reply:
done